css/templates/e-commerce/page.css

```diff
@@ -328,6 +332,14 @@ amp-selector [option][selected] {
   text-indent: -var(--space-2);
 }
 
+.ampstart-headerbar-fixed-link-wrapper {
+  width: 25px;
```

Comment:
wrap your svg with a div and  call it something like commerce-carti-icon-container and style it?
This sounds like it is a wrapper for the ampstart headerbar link section.